prime-angular-frontend/src/app/lib/classes/technical-supports-form-state.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { TechnicalSupportsFormState } from './technical-supports-form-state';
+
+describe('TechnicalSupportsFormState', () => {
+  it('should create an instance', () => {
+    expect(new TechnicalSupportsFormState()).toBeTruthy();
```

Comment:
it seems that a couple of FE tests are failing
<img width="1066" alt="image" src="https://user-images.githubusercontent.com/58282608/158891594-a2f81948-9e5d-4d7f-b30d-f3cdc854727a.png">